util/intellij.xml

```diff
@@ -0,0 +1,598 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was this an intentionally commited file?

Reply:
Not in this PR. It's the Google Code style. It could be nice to have.